src/Sulu/Bundle/TestBundle/Kernel/SuluTestKernel.php

```diff
@@ -106,7 +125,7 @@ public function registerContainerConfiguration(LoaderInterface $loader)
      */
     public function getCacheDir()
     {
-        return $this->rootDir . '/cache/' . $this->getContext() . '/' . $this->environment;
+        return $this->getProjectDir() . '/var/cache/' . $this->getContext() . '/' . $this->environment;
```

Comment:
`Path::join($this->getProjectDir(), 'var/cache', $this->getContext(), $this->environment)` https://github.com/webmozart/path-util

Reply:
no